Patches/Black Widows/ThingDefs_Weapons/Patch_Weapons.xml

```diff
@@ -54,7 +54,7 @@
 					  <graphicClass>Graphic_StackCount</graphicClass>
 					</graphicData>
 					<statBases>
-					  <MarketValue>0.52</MarketValue>
+					  <MarketValue>0.</MarketValue>
```

Comment:
Typo?

Reply:
Fixed all of your pointed out issues with the latest commit.